apps/redi-connect/src/pages/app/me/OnboardingSteps.tsx

```diff
@@ -0,0 +1,85 @@
+import { useMediaQuery, useTheme } from '@mui/material'
+import { ConnectProfileStatus } from '@talent-connect/data-access'
+import { Icon } from '@talent-connect/shared-atomic-design-components'
+import { useState } from 'react'
+import { Element } from 'react-bulma-components'
+import './OnboardingSteps.scss'
+import { STEPS } from './OnboardingStepsConstant'
+import HorizontalStepper from './stepper-card/HorizontalStepper'
+import VerticalStepper from './stepper-card/VerticalStepper'
+
+import { ReactComponent as StepDisabledImage } from '../../../../../../libs/shared-atomic-design-components/src/assets/images/step-disabled.svg'
+import { ReactComponent as StepDoneImage } from '../../../../../../libs/shared-atomic-design-components/src/assets/images/step-done.svg'
+import { ReactComponent as StepProgressImage } from '../../../../../../libs/shared-atomic-design-components/src/assets/images/step-progress.svg'
+
+const getCustomStepIcon = ({ active, completed, currentStep }) => {
+  if (active)
+    return currentStep === 3 ? <StepDoneImage /> : <StepProgressImage />
+
+  return completed ? <StepDoneImage /> : <StepDisabledImage />
+}
+
+const OnboardingSteps = ({
+  currentStep,
+  profile,
+  isMentor,
+  isCorporateMentor,
+}) => {
+  const [isStepperHidden, setIsStepperHidden] = useState(
+    localStorage.getItem('isStepperHidden') || false
```

Comment:
Nice work here. Though this exposes a weakness elsewhere:

When the user logs out, we run a `localStorage.clear()`. So when user logs back in again... the stepper will show up again.

Can you search for references to `localStorage` in the codebase and make sure we don't do a nuclear-bomb-style `localStorage.clear()`? Instead, when we need to clear a user session, we should instead call a more well-scoped function like `clearLocalStorageUserSessionData()` that **only** does what the name implies.

Reply:
Updated the code to use `localStorage.removeItem('postSalesforceMigrationAccessToken')` instead of `localStorage.clear()`.